rpm/xrootd-scitokens.spec

```
@@ -9,6 +9,9 @@ URL: https://github.com/scitokens/xrootd-scitokens
 # git archive v%{version} --prefix=xrootd-scitokens-%{version}/ | gzip -7 > ~/rpmbuild/SOURCES/xrootd-scitokens-%{version}.tar.gz
 Source0: %{name}-%{version}.tar.gz
 
+# Only on EL 7 and newer:
+Requires: gcc-c++ >= 4.8
```

```suggestion
BuildRequires: gcc-c++ >= 4.8
```
My bad, I should have said this in the first place.

---

Can you apply this change? Then we can merge it.